apstools/devices/positioner_soft_done.py

```diff
@@ -96,7 +97,7 @@ class PVPositionerSoftDone(PVPositioner):
     tolerance = Component(Signal, value=-1, kind="config")
     report_dmov_changes = Component(Signal, value=False, kind="omitted")
 
-    target = Component(Signal, value=-1, kind="config")
+    target = Component(Signal, value=UNDEFINED, kind="config")
```

Comment:
test for it!  in tests/test_positioner_soft_done.py

Reply:
Still unhappy:

```python
ValueError: Cannot determine the appropriate bluesky-friendly data type for value
<object object at 0x7f15b845cd20> of Python type <class 'object'>.
Supported types include: int, float, str, and iterables such as list, tuple, np.ndarray, and so on.
```

Reply:
what about np.nan?

Reply:
Aack!  Switch it back to `-1`.

Reply:
What about "None" (string)?

Reply:
Ok.  It should be an internal value that the user won't see until they forget to define it.

Reply:
This is only a problem until the positioner is moved for the first time